packages/engine/src/args.rs

```diff
@@ -50,6 +50,15 @@ pub struct Args {
     /// Logging output folder.
     #[clap(long, default_value = "./log")]
     pub log_folder: PathBuf,
+
```

Comment:
Spoke about it on slack, but for persistence, because it's the Engine arguments and not the CLI, we'll actually remove `global = true` from this arg. (Specifically there are no subcommands for the engine's cli so it's unnecessary)